packages/account-v2/src/components/CountrySelector/CountrySelector.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useResidenceList } from '@deriv/api';
+import { LabelPairedChevronDownMdRegularIcon } from '@deriv/quill-icons';
+import { Dropdown } from '@deriv-com/ui';
+
+type TCountrySelector = {
+    errorMessage?: React.ReactNode;
+    label: string;
```

Comment:
rEaCtNodE

Reply:
Cannot rEaCtNodE 🌚

Reply:
I get it cause of type mismatch from ui library.